hours/models.py

```diff
@@ -738,3 +738,16 @@ def apply_to_date_range(
         range_dates = set(expand_range(max_start_date, min_end_date))
 
         return matching_dates & range_dates
+
+
+class SignedAuthEntry(models.Model):
+    signature = models.TextField(verbose_name=_("Signature"))
+    created_at = models.DateTimeField(verbose_name=_("Signature created at"))
+    valid_until = models.DateTimeField(verbose_name=_("Signature valid until"))
+    invalidated_at = models.DateTimeField(
+        verbose_name=_("Invalidated time"), null=True, blank=True
```

Comment:
Do we want to store also non-validated signatures here at some point in the future? Then perhaps   `HaukiSignedAuthentication` should check that invalidated_at field is not null?

Reply:
We might want to yes. I thought we can change the check then. But might as well add it right now.